sources/editor/Xenko.GameStudio/DockingLayoutManager.cs

```diff
@@ -149,6 +149,10 @@ private void LoadLayoutFromSettings(MRUAdditionalData data, bool resetLayoutIfFa
             BindableSelectedItemsControl.DisableBindings = true;
             try
             {
+                // This exception is normal and will trigger a reset of the layout, since no layout can be loaded from the settings file.
+                if (GameStudioInternalSettings.CurrentLayoutVersion > data.DockingLayoutVersion)
```

Comment:
Can you adjust this check to `!=` rather than `>` please?

Reply:
Yes, sure!